### PR TITLE
VR-4666 Make sure tutorials have links to relevant APIs

### PR DESCRIPTION
**Pull request by nhatsmrt**
These are the ones I can find. Let me know if you have something else in mind.

### CODE REVIEW

File: client/verta/docs/tutorials/packaging/create_registered_model.rst [hunk 1]
```diff
@@ -8,15 +8,15 @@ This tutorial will go through how to create a registered model using the Client
 Using the client
 ----------------
 
-Registered models can be created as follows:
+Registered models can be created with :meth:`Client.create_registered_model() <verta.client.Client.create_registered_model()>` as follows:
```
Comment (convoliution) [inline client/verta/docs/tutorials/packaging/create_registered_model.rst]: This probably isn't a syntax error, but for consistency:
```suggestion
Registered models can be created with :meth:`Client.create_registered_model() <verta.client.Client.create_registered_model>` as follows:
```

File: client/verta/docs/tutorials/packaging/create_registered_model.rst [hunk 2]
```diff
@@ -8,15 +8,15 @@ This tutorial will go through how to create a registered model using the Client
...
 
 .. code-block:: python
 
     registered_model = client.create_registered_model(name="my model", labels=["research-purpose", "team-a"])
 
 Notice that we have assigned the labels ``"research-purpose"`` and ``"team-a"`` to the registered model. Labels are great tools for quickly filtering for a list of relevant registered models.
 
-Labels can also be added after the registered model has been created:
+Labels can also be added after the registered model has been created with :meth:`RegisteredModel.add_label() <verta._registry.model.RegisteredModel.add_label()>`:
```
Comment (convoliution) [inline client/verta/docs/tutorials/packaging/create_registered_model.rst]: See above
```suggestion
Labels can also be added after the registered model has been created with :meth:`RegisteredModel.add_label() <verta._registry.model.RegisteredModel.add_label>`:
```

**Review by convoliution — APPROVED** [review]
Minor comments; looks reasonable to me.

File: client/verta/docs/tutorials/packaging/create_registered_model.rst (hunk 2 above)
Comment (nhatsmrt) [reply client/verta/docs/tutorials/packaging/create_registered_model.rst]: Fixed.

File: client/verta/docs/tutorials/packaging/create_registered_model.rst (hunk 1 above)
Comment (nhatsmrt) [reply client/verta/docs/tutorials/packaging/create_registered_model.rst]: Fixed.

**Pull request merged**